lib/mixlib/install/generator/powershell/scripts/install_project.ps1

```diff
@@ -102,8 +104,19 @@ function Install-Project {
 }
 set-alias install -value Install-Project
 
-Function Install-ChefMsi($msi) {
-  $p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi" -Passthru -Wait
+Function Install-ChefMsi($msi, $addlocal) {
+  if ($addlocal -eq "service") {
+    $p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi ADDLOCAL=`"ChefServiceFeature`"" -Passthru -Wait
```

Comment:
@wrightp , I tried to use a hash key here:
```
$addlocal_str = @{"service" = "ADDLOCAL=\`"ChefServiceFeature\`""; "task" = "ADDLOCAL=`"ChefSchTaskFeature`""; "auto" = ""}

$p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi $addlocal_str[$addlocal]" -Passthru -Wait
```
In case of `auto` here `ADDLOCAL` parameter is not required, but if we go with the changes as mentioned above, an extra space gets added in `Start-Process` command after `$msi`(`$p = Start-Process -FilePath "msiexec.exe" -ArgumentList "/qn /i $msi " -Passthru -Wait`) and the command fails. So we can't skip at least one `If-Else` condition.
Also the grave accent character(\`) which is required before `"ChefServiceFeature"` and `"ChefSchTaskFeature"` creates issue while using this approach. It seems like it gets removed. Hence the command fails.